src/RealtimeEffectPanel.h

```diff
@@ -35,9 +36,15 @@ class RealtimeEffectPanel : public wxWindow
    ThemedButtonWrapper<wxBitmapButton>* mToggleEffects{nullptr};
    wxStaticText* mTrackTitle {nullptr};
    RealtimeEffectListWindow* mEffectList{nullptr};
-   wxWeakRef<AudacityProject> mProject;
+   AudacityProject& mProject;
```

Comment:
Replacing a weak pointer with a reference might look unsafe, but I think it is a safe enough assumption that this window will be destroyed only before the associated project.

Reply:
RealtimeEffectPanel is a child of ProjectWindow

Reply:
Right